src/components/DataTable.js

```diff
@@ -7,12 +7,25 @@ export default class DataTable extends React.Component {
     this.state = {
       page: 0,
       wide: false,
+      sort: {
+        column: '',
+        order: 'asc',
+      },
     };
     this.resize = this.resize.bind(this);
     this.prevPage = this.prevPage.bind(this);
     this.nextPage = this.nextPage.bind(this);
   }
 
+  componentWillMount() {
+    this.setState({
```

Comment:
Would `props.headers` in the constructor work? `this` isn't defined at that point, but `props` does get passed in as an argument.